docs/organization/integrations/source-code-mgmt/gitlab/index.mdx

```diff
@@ -12,6 +12,18 @@ Sentry owner, manager, or admin permissions and GitLab owner or maintainer permi
 
 </Alert>
 
+<Alert title="Note">
+  <p>
+  We recommend using a dedicated service account when installing this integration rather than a personal user account. 
+  </p>
+  <p>
+  Since the integration uses personal access tokens for authentication, all activities such as comments, work item creation, and commit associations will be attributed to the user who installed the integration. 
```

Comment:
Are these activities accurate for GitLab? I might be misremembering but I thought it just breaks if the user the organization, I don't recall commits being wrongly attributed.